js/d3-context-menu.js

```diff
@@ -161,14 +161,41 @@
 				if (openCallback.bind(element)(data, index) === false) {
 					return;
 				}
+        
+				//console.log(this.parentNode.parentNode.parentNode);//.getBoundingClientRect());   Use this if you want to align your menu from the containing element, otherwise aligns towards center of window
```

Comment:
Did you mean for this to be included in the PR?

Reply:
I wanted people to see how to align to a section of the screen instead of the window, if they needed that.  Perhaps it should be documented better.